components/sync15/src/telemetry.rs

```diff
@@ -547,6 +547,12 @@ impl SyncTelemetry {
     }
 }
 
+impl Default for SyncTelemetry {
```

Comment:
Can this be derived? I guess we'd have to implement `Default` for `Stopwatch`, too...